.github/workflows/daily-build-ballerina.yml

```
@@ -73,192 +189,253 @@ jobs:
+
+      - name: Slugify branch name
+        id: slug
+        run: echo "slug=$(echo '${{ matrix.vscodeBranch }}' | tr '/' '-')" >> "$GITHUB_OUTPUT"
+
+      - name: Download VSIX artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: ballerina-daily-vsix-${{ steps.slug.outputs.slug }}
+
+      - name: Notification - Ballerina
+        uses: ./.github/actions/dailyBuildNotification
```

actionlint (1.7.12)

[error] description is required in metadata of "Upload vsix to artifacts and notify" action at "/home/jailuser/git/.github/actions/dailyBuildNotification/action.yml"

(action)